packages/iTwinUI-react/src/core/Toast/Toaster.tsx

```diff
@@ -115,8 +107,40 @@ export default class Toaster {
     this.updateView();
   }
 
+  // Create container on demand.
+  // Cannot do it in constructor, because SSG/SSR apps would fail.
+  private async createContainer() {
+    const container = getContainer(TOASTS_CONTAINER_ID) ?? getDocument()?.body;
+    if (!container) {
+      return;
+    }
+
+    const toastWrapper = <ToastWrapper ref={this.toastsRef} />;
+
+    const _ReactDOM = ReactDOM as any; // eslint-disable-line @typescript-eslint/no-explicit-any
+
+    // v18 mode
+    if (_ReactDOM.createRoot) {
+      // suppress warning about importing createRoot from react-dom/client
+      _ReactDOM.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.usingClientEntryPoint = true;
```

Comment:
Don't we need to bring back original value after the import?

Reply:
Also, will they you be fired for this? 😯

Reply:
yes, thats the exact thing I was thinking whole night after making this change..
I think we need to set it to false, cause otherwise, all apps using iTwinUI will never see any warnings :D

Reply:
I reverted suppression just after we use `createRoot`...
Hopefully, that does not influence users app now

Reply:
Tested in Vite app by using `createRoot` in there and it showed warning

Reply:
Maybe we should leave the warning and ask users to suppress it if they complain?

Reply:
> Tested in Vite app by using `createRoot` in there and it showed warning

Maybe I misunderstood this one. Just want to double confirm, it shows a warning when `createRoot` from `react-dom` in client app and not when importing it from `react-dom/client`, correct?

And have you verified that the behavior remains the same with/without this new revert?

Reply:
> Maybe I misunderstood this one. Just want to double confirm, it shows a warning when `createRoot` from `react-dom` in client app and not when importing it from `react-dom/client`, correct?
> 
Yes, it just showed for importing in from wrong path in CLIENT APP (so not itwinui code)

> And have you verified that the behavior changes with/without this new revert? Without this revert, there was no warning in the user's app when importing `createRoot` from `react-dom`?
>

I have now and it shows no error :)
I mean.. if `createRoot` is called AFTER our toaster (cause only then we disable warning). So the fix there helps now.